atomics/T1044/T1044.yaml

```diff
@@ -0,0 +1,25 @@
+---
+attack_technique: T1044
+display_name: File System Permissions Weakness
+
+atomic_tests:
+- name: File System Permissions Weakness
+  description: |
+    This test to show checking file system permissions weakness and which can lead to privilege escalation by replacing malicious file. Example; check weak file permission and then replace.
+    powershell -c "Get-WmiObject win32_service | select PathName"   (check service file location) and 
+    copy /Y C:\temp\payload.exe C:\ProgramData\folder\Update\weakpermissionfile.exe   ( replace weak permission file with malicious file )
+
+  supported_platforms:
+    - windows
+
+  input_arguments:
+    weak_permission_file:
+      description: check weak files permission 
+      type: path
+      default: weak_permission_file.exe
+
+  executor:
+    name: powershell
+    command: |
+      get-acl "C:\ProgramData\folder\Update\#{weak_permission_file}" | FL | findstr "FullControl"
```

Comment:
Could you modify the definition of the test so it will work "out-of-the-box" on most systems? For example, maybe add command to create the weak_permission file and then use the path to that file in your command. This would allow the test to work by default even when users don't have a c:\ProgramData\folder\Update on their system.